File: client/src/app/dtos/memberDto.ts
import { ProjectDto } from "./ProjectDto";

export class memberDto {
    public id: number = 0;
    public UserName: string = '';
    public KnownAs: string = '';
    public Email: string = '';
    public Phone: string = '';
    public DateCreated!: Date;
    public DurationDays: number = 0;
    public Projects: ProjectDto[] = [];
}

File: API/DTOs/MemberDto.cs
using System.ComponentModel.DataAnnotations;

namespace API.DTOs
{
    public class MemberDto
    {
        public int Id { get; set; }
        public string UserName { get; set; }
        public string KnownAs { get; set; }
        [EmailAddress]
        public string Email { get; set; }
        [Phone]
        public string Phone { get; set; }
        public DateTime DateCreated { get; set; }
        public int DurationDays { get; set; }   
        public List<ProjectDto> Projects { get; set; }
    }
}